lib/read/setup.js

```diff
@@ -1,59 +1,78 @@
 const path = require('path');
 const fs   = require('fs');
-const os   = require('os');
-
+const yaml = require('js-yaml');
 const git  = require('simple-git');
-const Infra     = require('../infra/infrastructure');
-
 const Connector = require('infra.connectors');
 
+const Infra     = require('../infra/infrastructure');
+
 // Handles reading and processing setup block of stepfile
 class Setup {
 
-    async setup(doc, stepFile)
-    {
-        let provider = Object.keys(doc.setup)[0];
-        let cwd = provider === 'local' ? path.join(process.cwd(), path.dirname(stepFile), 'docable_results') : '.';
-
-        let docPath = path.dirname(stepFile);
-        // If using git repo, clone repo and update cwd/docPath
-        if( Object.keys(doc.setup).includes('git') )
-        {
-            console.log(`Cloning or pulling: ${doc.setup.git}`);
-            docPath = await this.cloneOrPull( doc.setup.git, path.dirname(stepFile) );
-            cwd = docPath;
+    /**
+     * Setup the env needed for execution
+     * @param {String} doc path to steps.yml to inline md file
+     * @param {String} html path to html, if doc is steps.yml
```

Comment:
Shouldn't this read, if doc _is not specified_ in steps.yml. 

What happens if there is a file in steps.yml and there is a html file provided. Which one gets precedence?

Reply:
This probably should read:
> @param {String} Override html file referenced in steps.yml. Otherwise, if no html file is specified in steps.yml, steps will be executed on provided file.

Reply:
addressed in 9ab4777.